### ISSUE TITLE
Add new maintainers

### ISSUE THREAD

**Issue by Loizzus** [rgossiaux/svelte-headlessui#179]
Hi there, 

As your package is popular but needs maintaining would you consider adding me and perhaps some others as maintainers of the project so we can allow pull requests, make changes, etc? 

Anything so we don't have to make another svelte headless UI package :)

Cheers,
Loic

**Comment by rgossiaux** [comment]
Yes, I think adding some other maintainers would be good for the project (I certainly haven't been maintaining it much and it deserves much more). I don't know anything about you specifically and I don't want to just add random strangers as maintainers without reason (nothing personal), but in general I'm completely open to the idea.

**Comment by Loizzus** [comment]
Fantastic! Let me know what you decide and we can coordinate with your chosen maintainers to get the fixes pushed through.

**Comment by rjruizes** [comment]
Loic has a decent online presence (Github, LinkedIn, personal website) so he seems like a good choice for a maintainer to me. I say this just as a developer who'd like to upgrade my dependencies.

**Comment by skinofstars** [comment]
I think you should either add a new maintainer or shut down the project. Quick scan of Loic looks like he'd be fine. You could just perhaps have a call with them and see?

**Comment by Loizzus** [comment]
Ok, I have made a new org for cases like this and added you all as maintainers so we can fix these issues as a team: https://github.com/PyongyangOpenSource/svelte-headlessui

**Comment by rjruizes** [comment]
@Loizzus You're going to have a hard time getting anyone to maintain a repo with that org name.

**Comment by Loizzus** [comment]
@rjruizes ah well, it was hard to find a name that wasn't already taken and that seemed funny.

**Comment by Conceptiks** [comment]
> Yes, I think adding some other maintainers would be good for the project (I certainly haven't been maintaining it much and it deserves much more). I don't know anything about you specifically and I don't want to just add random strangers as maintainers without reason (nothing personal), but in general I'm completely open to the idea.

If you are generally open to this, is there any way you could think of that would make you feel comfortable adding a maintainer to the project? For example, maybe you could ask on the official Svelte Discord for people to "apply" as maintainers? This of course would take a considerable amount of time on your side to vet the applicants, and I totally understand if your lack of time makes this impossible. I too would just be happy for this project to live on!

**Comment by sebastianfeistl** [comment]
Hello, I'm interested in using this package for a new project but would prefer it to be actively maintained. Adding new maintainers from the Svelte community could help ensure the project's continuity. This would be greatly appreciated by many developers, including myself.
